aws-vpn-gtw.tf

```diff
@@ -0,0 +1,10 @@
+# create aws vpn gateway for EKS VPC Frankfurt
+resource "aws_vpn_gateway" "vpn_gw" {
+  vpc_id = "${aws_vpc.frankfurt.id}"
+
+  tags = "${
+    map(
+     "Name", "eks aws vpn gateway frankfurt"
```

Comment:
Can we take all the regions as a parameter? Not hardcode with Frankfurt or eu-east-1 and so on.